golem-common/src/model/plugin.rs

```diff
@@ -282,45 +279,6 @@ impl PluginScope {
             Self::Project(scope) => &scope.project_id == project_id,
         }
     }
-
-    pub async fn accessible_scopes(
```

Comment:
Why isn't this necessary anymore?

Reply:
This is inline in the AuthedComponentService